code/modules/recycling/conveyor2.dm

```diff
@@ -277,24 +286,28 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	else
 		icon_state = "switch-off"
 
-
-// timed process
 // if the switch changed, update the linked conveyors
-
-/obj/machinery/conveyor_switch/process()
-	if(!operated)
-		return
-	operated = 0
-
+/obj/machinery/conveyor_switch/proc/update_linked_conveyors()
 	for(var/obj/machinery/conveyor/C in GLOB.conveyors_by_id[id])
 		C.operating = position
 		C.update_move_direction()
 		C.update_icon()
+		if(abs(C.operating)) // equals 1 if the conveyor is moving in either direction, 0 if turned off
```

Comment:
Both `-1` and `1` are truthy values. `abs()` makes sense but it's not really needed.